language_dump.py

```diff
@@ -63,10 +69,18 @@ def dump_translations():
     fallback_language = args.fallback
     verbose = args.verbose
     strings_by_object = {}
+    reference_str_count = 0
+    translated_str_count = 0
 
     for filename in glob.iglob(args.objects + '/**/*.json', recursive=True):
-        extract_translations(language_to_extract, fallback_language, verbose,\
-                             filename, strings_by_object)
+        obj_translations, ref_obj_translation =\
+            extract_translations(language_to_extract, fallback_language, verbose,\
+                                 filename, strings_by_object)
+        reference_str_count += ref_obj_translation
+        translated_str_count += obj_translations
+
+    translation_progress = round(100*translated_str_count/reference_str_count, 2)
```

Comment:
```suggestion
    translation_progress = round(100 * translated_str_count / reference_str_count, 2)
```

Are you using pylint? I'd imagine this kind of thing would be covered by it.

Reply:
I'm it actually doesn't suggest this.